routing/graph.go

```diff
@@ -6,9 +6,9 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
-// routingGraph is an abstract interface that provides information about nodes
+// Graph is an abstract interface that provides information about nodes
 // and edges to pathfinding.
-type routingGraph interface {
+type Graph interface {
```

Comment:
why do we want to export this interface when its implementation details are not exported?